integration/docker/cgroups_test.go

```diff
@@ -98,9 +134,13 @@ func checkCPUCgroups(name string, expected expectedCPUValues) error {
 			return err
 		}
 
+		if SandboxCgroupOnly {
+			fmt.Fprintf(GinkgoWriter, "PodCgroupOnly enabled, cgroup is managed by caller, will not check values")
```

Comment:
s this a `Skip` ?

Reply:
Added a comment why, it just do a continue, probably make sense to split the test if can read and the value is good, I just adding continue now.